iosxr_test.py

```
@@ -89,8 +92,8 @@ def bringup_vagrant():
         s = pexpect.pxssh.pxssh()
         s.login(hostname, username, password, terminal_type, linux_prompt, login_timeout, port)
     except pxssh.ExceptionPxssh, e:
-        print("pxssh failed on login")
-        print(str(e))
+        logger.debug("pxssh failed on login")
+        logger.debug(str(e))
```

Is it really the desired behavior to fail silently unless debugging is enabled?